### PR TITLE
reformatting the binary cross entropy loss

### PR DESCRIPTION
**Pull request by anashas**
References #1083

### CODE REVIEW

File: ivy/functional/ivy/losses.py [hunk 1]
```diff
@@ -53,22 +53,67 @@ def cross_entropy(
...
+
+    Instance Method Examples
+    ------------------------
+
+    Using :code:`ivy.Array` instance method:
+
+    >>> x = ivy.array([1, 0, 0, 0])
+    >>> y = ivy.array([0.8, 0.2, 0.2, 0.2])
+    >>> print(ivy.binary_cross_entropy(x, y))
+    ivy.array([0.2231, 0.2231, 0.2231, 0.2231]))
```
Comment (bhattbhuwan13) [inline ivy/functional/ivy/losses.py:116]: There is an extra parenthesis at the end here

Comment (anashas) [reply ivy/functional/ivy/losses.py]: Hi @bhattbhuwan13, I just deleted the extra parenthesis

**Comment by anashas** [comment]
Hi @bhattbhuwan13 , can you please resolve the merge conflict?

**Comment by bhattbhuwan13** [comment]
Hi @[anashas](https://github.com/anashas), I am not responsible for merging the code. I saw a mistake and pointed it out to you. That's it.

**Comment by anashas** [comment]
Hi @bhattbhuwan13, thanks for the response, it's okay I fixed the merge conflict. Thank you for your time

**Pull request merged**